### ISSUE TITLE
Feature request: Paste around selection

### ISSUE THREAD

**Issue by lucasvieites** [Clipy/Clipy#284]
Hi,
this feature would enable you to have a "variable" in your snippet which would be replaced with the selected text you are pasting, for example"

1.  your snippet contains a predefined variable text: "```<a href='%THEVARIABLE%'>%THEVARIABLE</a>```"
2. you select a piece of text, say: "```https://www.github.com```"
3. you paste the snippet "over" the selection and the variable gets replaced
4. result: "```<a href='https://www.github.com'>https://www.github.com</a>```"

**Comment by tiggeymone** [comment]
Any news on this?

**Comment by lucasvieites** [comment]
Nope, nothing yet. In the meantime, I'm using [Espanso](https://espanso.org/), which has this feature, and many more.
